shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -554,7 +568,8 @@ public static WireCommand readFrom(DataInput in, int length) throws IOException
         final ByteBuf data;
         final int numEvents;
         final long lastEventNumber;
-        final long unused; // Will be used by AppendSequence:  
+        final long unused; // Will be used by AppendSequence:
+        final long requestId;
```

Comment:
I think you can take over the unused long, conditional appends went a different way.

Reply:
ok.